dotcom-rendering/src/web/components/ClickToView.tsx

```diff
@@ -89,15 +91,26 @@ const shouldDisplayOverlay = ({
 	isTracking,
 	isOverlayClicked,
 	isMainMedia,
+	format,
 }: {
 	isTracking: boolean;
 	isOverlayClicked: boolean;
 	isMainMedia?: boolean;
+	format: ArticleFormat;
 }) => {
-	if (isMainMedia || !isTracking) {
+	if (isMainMedia) {
 		return false;
 	}
-	if (isOverlayClicked) {
+
+	// See https://github.com/guardian/frontend/issues/25454
+	// This dependency on format can be removed once CAPI is properly setting the tracking field correctly.
+	if (
+		format.design === ArticleDesign.LiveBlog ||
+		format.design === ArticleDesign.DeadBlog
+	) {
+		return true;
+	}
```

Comment:
I think this will prevent the overlay from ever being clicked away

Reply:
Reminder to self: Make sure things are still interactive and don't just look good. Thanks Oliver!